command/src/channel.rs

```diff
@@ -585,27 +576,28 @@ mod tests {
             Channel<Serializable, Serializable>,
             Channel<Serializable, Serializable>,
         ) = Channel::generate(1000, 10000).expect("Could not create nonblocking channel");
-        writing_channel.blocking();
+        writing_channel.blocking().unwrap();
 
         println!("reading message in a detached thread, with a timeout of 200 milliseconds...");
         let awaiting_with_timeout = thread::spawn(move || {
-            let message =
-                reading_channel.read_message_blocking_timeout(Some(Duration::from_millis(200)));
+            let message = reading_channel
+                .read_message_blocking_timeout(Some(Duration::from_millis(200)))
+                .unwrap();
             println!("read message!");
             message
         });
 
         println!("Waiting 100 milliseconds…");
         thread::sleep(std::time::Duration::from_millis(100));
 
-        writing_channel.write_message(&Serializable(100));
+        writing_channel.write_message(&Serializable(100)).unwrap();
```

Comment:
To me, this should be replaced by an `expect`

Reply:
done